feature-group-definitions/array-group.yml

```diff
@@ -1,4 +1,13 @@
 spec: https://tc39.es/proposal-array-grouping/
+status:
+  baseline: false
+  support:
+    chrome: "117"
+    chrome_android: "117"
+    edge: "117"
+    firefox: "119"
+    firefox_android: "119"
+    safari: preview
 compat_features:
-  - javascript.builtins.Array.group
```

Comment:
Oh, oops :) https://tc39.es/proposal-array-grouping/ agrees with the changes.

Reply:
This wasn't a mistake, at least at the time. It changed names rather recently.